ui/packages/catalog-ui-search/src/main/webapp/component/search-form-editor/search-form-editor.hbs

```diff
@@ -0,0 +1,20 @@
+{{!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ --}}
+<div style="width: 100% overflow: hidden">
+<div style="width: 37rem; height: 57rem; float: left">
+<div class="search-form-editor-title" style="float: left; padding-left: 3rem;"></div>
```

Comment:
Do we generally use inline style for something like this, or should it be moved out into a CSS file?